pkgs/applications/misc/ulauncher/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub, python27Packages, substituteAll,
```

Comment:
Maybe also add a comment that Python 3 is not supported with a link to https://github.com/Ulauncher/Ulauncher/pull/279

Reply:
Yeah almost forgot that.

We should also add 
```
disabled = ! isPy27;
```

and it will hard fail if someone tries to build it for those interpreter versions.